src/app/api/gql/scheduling-dataset/derivative-batching.graphql

```diff
@@ -0,0 +1,20 @@
+query getDatasetBatching($datasetId: DatasetID!) {
```

Comment:
Very bad name

Reply:
Do not hardcode flow type, make it a parameter.
Do not mention "derivative" in the file name, batching condition might be possible for other flow types in future, and the model is wider then the expectations of "EXECUTE_QUERY".

Reply:
Ok, added one `datasetFlowType` parameter